__init__.py

```diff
@@ -1150,6 +1151,38 @@ def _speak_weather(self, dialog):
         self.log.info("Speaking dialog: " + dialog.name)
         self.speak_dialog(dialog.name, dialog.data, wait=True)
 
+    @skill_api_method
+    def get_current_weather(self, message=None):
+        """Get the current temperature and condition code.
+        
+        Args:
+            message (Message): [Optional] mock message will be parsed to
+                               determine weather intent.
+
+        Returns:
+            Dict: {
+                temperature: current temperature
+                high_temperature: forecasted high for today
+                low_temperature: forecasted low for today
+                condition_code: code representing overall weather condition
+                                see Maps for all codes in skill/weather.py
+            }
+        """
+        self.log.debug("Requested current weather information:")
+        if message is None:
+            message = Message("", data={"utterance": ""})
+        intent_data = self._get_intent_data(message)
+        weather = self._get_weather(intent_data)
+        data = {
+            "temperature": weather.current.temperature,
+            "high_temperature": weather.current.high_temperature,
+            "low_temperature": weather.current.low_temperature,
+            "condition_code": weather.current.condition.code
+        }
+        for k in data:
```

Comment:
How often will this get called?  Do we really want debug messages every time this is called?

Reply:
Yeah it will be very regularly. Chose debug for this reason, but happy to remove them.

My thinking was beyond the Homescreen Skill - if a Skill developer is trying to use it in another Skill and they aren't sure whether it's working or not, this helps them to know if it's being called and what data it's sending back. Figured this might be one used particularly by newer devs.

Reply:
removed